### PR TITLE
Add a connection status display to the tiny example. Also improve style a bit

### PR DESCRIPTION
**Pull request by brianpeiris**

### CODE REVIEW

File: client/tiny.js [hunk 1]
```diff
@@ -19,16 +19,23 @@ var c = {
...
       signal.plugindata &&
       signal.plugindata.data &&
       signal.plugindata.data.success === false;
   return isPluginError || Minijanus.JanusSession.prototype.isError(signal);
-};
+}
 
 function init() {
-  var ws = new WebSocket("ws://localhost:8188", "janus-protocol");
+  const server = `wss://${location.hostname}:8989`;
```
Comment (brianpeiris) [inline client/tiny.js]: Maybe we can make this configurable in the query string if you prefer it to be localhost by default.

File: client/tiny.js [hunk 2]
```diff
@@ -160,18 +167,23 @@ function attachPublisher(session) {
...
+        showStatus(`Joined room ${roomId}`);
         var occupants = reply.plugindata.data.response.users[roomId];
         for (var i = 0; i < occupants.length; i++) {
           if (occupants[i] !== USER_ID) {
             addUser(session, occupants[i]);
           }
         }
       })
-      .then(() => { return { handle, conn, channel, unreliableChannel }; });
+      .then(() => { handle, conn, channel, unreliableChannel });
```
Comment (brianpeiris) [inline client/tiny.js]: Not sure if you have a strong preference for the previous syntax.

File: client/tiny.js (hunk 1 above)
Comment (mquander) [reply client/tiny.js]: Query string or textbox on the page is best IMO. I don't want to screw up by forgetting which server a given tab is connecting to, so I want it to be visible.

File: client/tiny.js (hunk 2 above)
Comment (mquander) [reply client/tiny.js]: Don't care.

**Comment by mquander** [comment]
👍

**Pull request merged**